core/src/main/java/org/apache/iceberg/rest/requests/FetchScanTasksRequest.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iceberg.rest.requests;
+
+import org.apache.iceberg.relocated.com.google.common.base.MoreObjects;
+import org.apache.iceberg.relocated.com.google.common.base.Preconditions;
+import org.apache.iceberg.rest.RESTRequest;
+
+@SuppressWarnings("checkstyle:VisibilityModifier")
```

Comment:
Do we need this?

Reply:
Let me see if I can get rid of this, this was for not having getter and setters for the the `plan-task` I believe.

Reply:
<img width="861" alt="Screenshot 2024-11-27 at 11 31 23 AM" src="https://github.com/user-attachments/assets/67e57d0f-8a63-44bd-819f-55e71ea082d3">


Im not sure if checkStyle has a bug or if im doing something incorrect. It seems that checkStyle is complaining that the private var `planTask` does not have an accessor method. However I do have the following in the class

```
public String planTask() {
    return planTask;
  }
```

And it seems to follow the pattern such as https://github.com/apache/iceberg/blob/main/core/src/main/java/org/apache/iceberg/rest/requests/UpdateTableRequest.java

If i remove the `@SuppressWarnings("checkstyle:VisibilityModifier")` it will continue to fail the build unfortunately so will keep this for now.